specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2022-10-01/vaultusages.json

```diff
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-10-01",
+    "title": "RecoveryServicesClient",
```

Comment:
Is not clear what new changes are being introduced with this API version.

The first commit needs to be the previous api version and the new changes should only be added in the subsequent commits. This is explained in detail here (If using Open API which is recommended)  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version) or here(For manual process) :  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version) . Please remove the "ARMChangesRequested" label once you have fixed this issue.

Reply:
Hi @rkmanda, thank you so much for the review.

1) The first commit is indeed generated by the [OpenAPI Hub](https://aka.ms/openapihub).
![image](https://user-images.githubusercontent.com/4202918/201610160-5424e732-5188-47e1-a3ed-6c3e1bf609ff.png)

2) There are no significant changes because it's just for complaint. The actual changes were done in `recoveryservicessiterecovery` PR: https://github.com/Azure/azure-rest-api-specs/pull/20998

This PR along with `recoveryservicesbackup` are just for compliance.

3 PRs: https://github.com/Azure/azure-rest-api-specs/pulls/shesamian